compiler/plugins/target/ROCM/Dialect/ROCM/Transforms/test/apply_builtin_ukernel_pdl_patterns.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: iree-opt --pass-pipeline='builtin.module(func.func(iree-rocm-apply-builtin-pdl-patterns{targets=gfx942 enable-tensor-ukernels=true}))' \
+// RUN: iree-opt --pass-pipeline='builtin.module(func.func(iree-rocm-apply-builtin-pdl-patterns{targets=gfx942,gfx950 enable-tensor-ukernels=true}))' \
```

Comment:
I think f8E4M3FNUZ patterns shouldn't match if we set the targets to both gfx942 and gfx950 as that's gfx942 only? I would have expected an additional RUN command here with a different prefix like:
```
iree-opt --pass-pipeline='builtin.module(func.func(iree-rocm-apply-builtin-pdl-patterns{targets=gfx942,gfx950 enable-tensor-ukernels=true}))' --mlir-print-local-scope --split-input-file %s | FileCheck %s --check-prefix=GFX950
``` 
And then in the checks I think we should see something like:
```
// CHECK-SAME: iree_codegen.ukernel = #iree_codegen.ukernel_descriptor<"pingpong_medium_f8E4M3FNUZ_expanded", tensor>
// GFX950-SAME: iree_codegen.ukernel = #iree_codegen.ukernel_descriptor<"pingpong_medium_f8E4M3FN_expanded", tensor>
```